firebase-ml-modeldownloader/src/main/java/com/google/firebase/ml/modeldownloader/internal/ModelFileManager.java

```diff
@@ -61,6 +63,24 @@ public static ModelFileManager getInstance() {
     return FirebaseApp.getInstance().get(ModelFileManager.class);
   }
 
+  void deleteNonLatestCustomModels() throws FirebaseMlException {
+    File root = getDirImpl("");
+    // for each custom model sub directory
+
+    boolean ret = true;
+    if (root.isDirectory()) {
+      for (File f : Preconditions.checkNotNull(root.listFiles())) {
```

Comment:
This will cause the app to crash if listFiles ever returns null. Is that intended?

Reply:
Good catch - adding fix in second location as well.